server/src/controllers/asset.controller.ts

```diff
@@ -197,4 +198,42 @@ export class AssetController {
   deleteAssetMetadata(@Auth() auth: AuthDto, @Param() { id, key }: AssetMetadataRouteParams): Promise<void> {
     return this.service.deleteMetadataByKey(auth, id, key);
   }
+
+  @Put(':id/edits')
+  @Authenticated({ permission: Permission.AssetEdit })
+  @Endpoint({
+    summary: 'Applies edits to an existing asset',
+    description: 'Applies a series of edit actions (crop, rotate, mirror) to the specified asset.',
+    history: new HistoryBuilder().added('v2').beta('v2'),
```

Comment:
All of these `v2` versions should eventually be replaced with the actual version it gets released in, like `v2.5.0`.